ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/SqlToJavaVisitor.java

```diff
@@ -1195,6 +1198,25 @@ private String evaluateOrReturnNull(final String s, final String type) {
       }
     }
 
+    private String evaluateOrReturnNull(final String s, final String type, final String javaType) {
```

Comment:
The other `evaluateOrReturnNull` could be updated to just return `evaluateOrReturnNull(s, type, "Object")`

Reply:
I was thinking about that, and did that initially.

I'm partially worried that casts like that would cause some slight performance overhead.  I suppose JMH microbenchmarks could (try and) show whether or not that's an issue.

If you aren't worried about it, I can switch to that approach.  If you agree with me, then I should probably add a comment to explain why the code is doing something obviously repetitive.

Reply:
Oh true. That's unfortunate. Yeah let's keep it like this then.